docs/contribute/configuration.md

```diff
@@ -27,194 +27,208 @@ page_nav:
-
-<p>Comments are available only on <code class="highlighter-rouge">default</code> page layout and you have to enable them on new pages with <code class="highlighter-rouge">comments: true</code> variable.</p>
-
-<p> Currently, we have integrated Discourse comments into our SingularityNET Developer Portal which link directly to topics created on <a href="https://community.singularitynet.io/c/developers" target="_blank">our forum</a>.
-
-<div class="example"></div>
-<div class="language-yml highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="na">snet</span><span class="pi">:</span>
-<span class="na">comments</span><span class="pi">:</span>
-<span class="na">disqus_forum_shortname</span><span class="pi">:</span>
-</code></pre></div></div>
-
-<div class="callout callout--warning">
-<p><strong>Build site with environment variable!</strong> When you run <code>jekyll serve</code>, your Disqus commenting system shouldn’t render at all in any pages.</p>
-<p>We’ll set Jekyll to only render Disqus commenting system when its environment is set to production.</p>
-<p>So then, how do you get the Disqus commenting system to only show up on a production environment? When building your Jekyll project with jekyll build, you’ll want to prefix it with <code>JEKYLL_ENV=production</code> so the complete command looks like this one: <code>JEKYLL_ENV=production jekyll build</code></p>
-</div>
-
-<h2 id="favicon">Favicon</h2>
-<p>To change favicon just replace <code class="highlighter-rouge">/favicon.ico</code> with your new icon. Make sure it is in <code class="highlighter-rouge">.ico</code> format. Dimensions should be 16px x 16px.</p>
-
-<div class="callout callout--info">
-<p><strong>Favicon <code>.psd</code> file included!</strong> We've included <code>.psd</code> file with pre-made favicon in <code>/designs</code> folder of your theme.</p>
-</div>
+## Introduction
 
+`_config.yml` stores configuration data. Many of these options can be specified from the command line executable but it’s easier to specify them here so you don’t have to remember them.
+
+>__Please stop and re-run `jekyll serve` command after you change configuration file!__ <br> Master configuration file contains global configurations and variable definitions that are read once at execution time. Changes made to `config.yml` during automatic regeneration are not loaded until the next execution.
+
+## Relative URLs
+
+If you’re deploying to server where your site is not going to be in `root` directory, you should setup `baseurl` variable.
+
+For example, if your site is going to be stored on URL that looks like this `http://example.com/project`, you’ll have to update your `baseurl` variable and it should look like this:
+
+```
+snet:
```

Comment:
these config stanzas are meant to be idented (compare to the repos `.config.yml`)

Reply:
Done